backend/routes/indexer.go

```diff
@@ -165,6 +168,51 @@ func processPixelPlacedEvent(event IndexerEvent, w http.ResponseWriter) {
 	WriteResultJson(w, "Pixel placement indexed successfully")
 }
 
+func processExtraPixelsPlacedEvent(event IndexerEvent, w http.ResponseWriter) {
+	placedByAddress := event.Event.Keys[1][2:]
+	positionsLenHex := event.Event.Data[0]
+
+	positionsLen, err := strconv.ParseInt(positionsLenHex, 0, 64)
+	if err != nil {
+		WriteErrorJson(w, http.StatusInternalServerError, "Error converting positionsLen hex to int")
+		return
+	}
+
+	colorsLenHex := event.Event.Data[1+positionsLen]
+	colorsLen, err := strconv.ParseInt(colorsLenHex, 0, 64)
+	if err != nil {
+		WriteErrorJson(w, http.StatusInternalServerError, "Error converting colorsLen hex to int")
+		return
+	}
+	if colorsLen != positionsLen {
+		WriteErrorJson(w, http.StatusBadRequest, "Colors length does not match positions length")
+		return
+	}
+
```

Comment:
We will want to carry out the same logic as `placedPixelEvent` here to add to postgres and redis since we will only be going off of this event.